### PR TITLE
Update net-istio controller name

### PR DESCRIPTION
**Pull request by nak3**
Image is replaced by `IMAGE_<deployment name>`.

This patch updates the <deployment name> from `networking-istio` to `net-istio-controller`.

### CODE REVIEW

**Comment by nak3** [comment]
/test 4.8-upstream-e2e-mesh-aws-ocp-48

infra issue

```
msg=failed to fetch Cluster: failed to generate asset "Cluster": failed to create cluster: failed to apply Terraform: failed to complete the change
```

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve
/hold

Please unhold once all mesh tests are good.

**Comment by nak3** [comment]
/hold cancel

Passed :tada: 
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift-knative_serverless-operator/1160/pull-ci-openshift-knative-serverless-operator-main-4.8-upstream-e2e-mesh-aws-ocp-48/1430043514980274176

File: openshift-knative-operator/hack/007-networkpolicy-mesh.patch [hunk 1]
```diff
@@ -22,15 +22,15 @@ index 00000000..3d46fbed
 +apiVersion: networking.k8s.io/v1
 +kind: NetworkPolicy
 +metadata:
-+  name: istio-webhook
```
Comment (nak3) [inline openshift-knative-operator/hack/007-networkpolicy-mesh.patch:25]: Note for myself. This old networkpolicy also needs to be deleted by operator :sweat:

**Pull request merged**